.github/workflows/ci-test-incluster.yml

```diff
@@ -0,0 +1,79 @@
+name: run-in-cluster-test
+
+on:
+  push:
+    branches:
+    - "**"
+    paths:
+    - "deployments/annotations/**"
+    - "deployments/generic/**"
+    - "tests/test-scenarios-github.sh"
+    - ".github/workflows/ci-test-incluster.yml"
+  pull_request:
+    branches: ["*"]
+    paths: 
+    - "deployments/annotations/**"
+    - "deployments/generic/**"
+    - "tests/test-scenarios-github.sh"
+    - ".github/workflows/ci-test-incluster.yml"
+
+jobs:
+  manifest-test:
+    name: Run basic manifest tests / ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, ubuntu-18.04]
+    steps:
+      - name: Kernel version
+        run: uname -r
+      
+      - uses: actions/checkout@v2
+
+      - name: Setup Enviroment
+        run: |
+          ./contribution/k3s/install_k3s.sh
+
+      - name: Install cmctl
+        run: |
+          OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -sSL -o cmctl.tar.gz https://github.com/cert-manager/cert-manager/releases/download/v1.7.2/cmctl-$OS-$ARCH.tar.gz
+          tar xzf cmctl.tar.gz
+          sudo mv cmctl /usr/local/bin
+
+      - name: Install annotation controller
+        run: |
+          kubectl apply -f deployments/annotations/cert-manager.yaml
+          kubectl wait pods --for=condition=ready -n cert-manager -l app.kubernetes.io/instance=cert-manager
+          cmctl check api  --wait 300s
+          kubectl apply -f deployments/annotations/kubearmor-annotation-manager.yaml
+          kubectl wait pods --for=condition=ready -n kube-system -l kubearmor-app=kubearmor-annotation-manager
+      
+      - name: Apply KubeArmor manifest
+        run: |
+            kubectl apply -f deployments/generic/kubearmor.yaml
```

Comment:
Can we use the YAML in build/ ? The generic yaml is configured to use stable.

Also we would have empty artificats on failure,
Can refer to https://github.com/daemon1024/KubeArmor/blob/bcc-to-libbpf/.github/workflows/ci-test-runtime.yml
Which I am using to test #677

Reply:
No, this workflow is to make sure that changes to yaml are valid and wont impact kubearmor or any of it's components.